stdlib/REPL/src/Terminals.jl

```diff
@@ -169,6 +171,31 @@ else
     end
 end
 
+function has_light_background(t::TTYTerminal)
+    # Don't bother if there is no support for colors
+    !hascolor(t) && return false
+    # See
+    # <https://unix.stackexchange.com/questions/245378/common-environment-variable-to-set-dark-or-light-terminal-background>
+    # for how to detect the terminal background color
+    colorfgbg = get(ENV, "COLORFGBG", "")
+    sep = findlast(";", colorfgbg)
+    if sep !== Nothing
+        # See e.g. <https://en.wikipedia.org/wiki/ANSI_escape_code>
+        # for the 16 possible colors
+        colorbg = tryparse(Int, colorfgbg[last(sep)+1:end])
+        if colorbg !== Nothing && colorbg >= 0 && colorbg <= 15
+            return colorbg == 7 || colorbg >= 9
+        end
+    end
+    # (We could/should use more mechanisms here)
+    # Blindly guess an OS-dependent default
+    if Sys.isapple()
+        return true
```

Comment:
This is a bad assumption. iTerm2 defaults to black. However, iTerm2 does have color query escape codes.

Reply:
To my knowledge, iTerm2 sets the `COLORFGBG` environment variable that we check above, thus this default isn't reached.

Reply:
I don't think we should ever default this to true solely based on OS.

Reply:
I removed the default based on OS; all OS now use the same default